src/ex1/script.js

```diff
@@ -0,0 +1,68 @@
+// trigger fork
+let count = 0;
+
+function addTodo(){
+    const todo = document.getElementById("newTodo").value;
+    if(todo.length === 0){
+        alert("Please enter a todo task")
+    }
+    else {
+        const todoList = document.getElementById("todoList");
+        const li = document.createElement("li");
+        li.id = 'todo' + count;
+        li.innerHTML = `<input type='checkbox' name='selectTodo' class= pl value="${todo}">`;
+        li.innerHTML += todo;
+        todoList.appendChild(li);
+        document.getElementById("newTodo").value = "";
+        count++;
+        console.log("is this checked" + li.checked);
+    }
+}
+document.getElementById("addTodo").addEventListener("click", addTodo);
+
+
+let history = [];
```

Comment:
```suggestion
const history = [];
```
You only push stuff to the history, you don't reassign it, so you can use const